Makefile

```diff
@@ -1,15 +1,16 @@
 IMG ?= flagd:latest
 PHONY: .docker-build .build .run .mockgen
 PREFIX=/usr/local
+VERSION=0.0.8
```

Comment:
We can have release please automatically bump this version 😄

Reply:
For example: https://github.com/open-feature/node-sdk-contrib/blob/3bc87990f3db5dae27969b5eaa9c1ee77dbb7ef3/libs/hooks/open-telemetry/src/lib/open-telemetry-hook.ts#L23

Reply:
Will this need to be maintained manually?

Reply:
I could have it increment in the release job if you want me to add that into this PR?

Reply:
I think we need to automate this or we'll inevitably forget about it in the future.

Reply:
Well, it's also the same with OFO. 
I would like to do them at the same time, I'll open an issue after this to track

Reply:
Or we could base the version off the current /ref/tag which probably would give the same outcome.

Reply:
automated this @beeme1mr